cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -34,7 +34,7 @@ spec:
     count: 3
     allowMultiplePerNode: false
   mgr:
-    modules:
+    modules: []
```

Comment:
Rather than add the empty array, how about if we comment out the `modules` line? Otherwise if they uncomment the pg_autoscaler, it will be broken unless they remove the brackets.

Reply:
done, but I don't know if it is better this way.